examples/property_estimator/start_client.py

```diff
@@ -8,22 +8,43 @@
 from propertyestimator.utils import get_data_filename, setup_timestamp_logging
 
 
-def submit_calculation_to_server():
+def request_estimate_blocking():
     """Submit calculations to a running server instance"""
+    setup_timestamp_logging()
+
+    # Load in the data set of interest.
+    data_set = ThermoMLDataSet.from_file_list(get_data_filename('properties/single_density.xml'))
+    # Load in the force field to use.
+    force_field = smirnoff.ForceField(get_data_filename('forcefield/smirnoff99Frosst.offxml'))
+
+    # Create the client object.
+    property_estimator = client.PropertyEstimatorClient()
+    # Submit the request to a running server, and wait for the results.
+    result = property_estimator.estimate(data_set, force_field)
 
+    logging.info('The server has returned a response: {}'.format(result))
+
+
+def request_estimate_non_blocking():
```

Comment:
Maybe the non-blocking call could be `compute_estimate_asynchronous()`, `compute_estimate_async()`, `estimate_asynchronous()`, or `estimate_async()`?

We might also draw some inspiration for idioms from the [Python asyncio module](https://docs.python.org/3/library/asyncio.html).

Reply:
Alternatively, you can just have an `.estimate()` method and add an optional argument `asynchronous=False` by default, with the user able to specify `asynchronous=True` to get an asynchronous estimate.

Reply:
The synchronous= route is the way I updated it to a few minutes ago per @andrrizzi's suggestion, and I think that looks pretty clean.